### PR TITLE
Updates for new getspeech webhook properties

### PR DESCRIPTION
**Pull request by wajiw**

### CODE REVIEW

File: src/test/java/com/vailsys/freeclimb/webhooks/percl/GetSpeechActionCallbackTest.java [hunk 1]
```diff
@@ -8,7 +8,7 @@
...
     private GetSpeechActionCallback gsac;
-    private static final String GSAC_JSON= " { \"requestId\":\"RA1766ca5ee92fc6c528b72aff5e8b48f4e056e8\", \"requestType\":\"getSpeech\", \"callId\":\"CAbde0362aef3d228b3a39baafa9e4f0204e724966\", \"accountId\":\"ACae05391ecca1352e9108d545482a1e6f384e7a49\", \"from\":\"+17083168669\", \"to\":\"+12248806211\",\"callStatus\":\"completed\", \"direction\":\"inbound\", \"conferenceId\":null, \"queueId\":null, \"callerInfo\":null, \"reason\":\"recognition\", \"recognitionResult\":\"yellow\", \"confidence\":55 }";
+    private static final String GSAC_JSON= " { \"requestId\":\"RA1766ca5ee92fc6c528b72aff5e8b48f4e056e8\", \"requestType\":\"getSpeech\", \"callId\":\"CAbde0362aef3d228b3a39baafa9e4f0204e724966\", \"accountId\":\"ACae05391ecca1352e9108d545482a1e6f384e7a49\", \"from\":\"+17083168669\", \"to\":\"+12248806211\",\"callStatus\":\"completed\", \"direction\":\"inbound\", \"conferenceId\":null, \"queueId\":null, \"callerInfo\":null, \"reason\":\"recognition\", \"recognitionResult\":\"yellow\", \"confidence\":55, \"completionReason\":\"test\", \"completionCause\":\"test\", \"mrcpCode\":55, \"mrcpDiagnostic\":\"test\" }";
```
Comment (ngarg-vail) [inline src/test/java/com/vailsys/freeclimb/webhooks/percl/GetSpeechActionCallbackTest.java:11]: For readability could we make this a multi-line string? - https://www.baeldung.com/java-multiline-string
that should make the JSON string easier understand & modify in the future (shouldn't have to deal with any quote escaping)

**Review by ngarg-vail — CHANGES_REQUESTED** [review]
@wajiw Do we need changelog updates in this?

File: src/test/java/com/vailsys/freeclimb/webhooks/percl/GetSpeechActionCallbackTest.java (hunk 1 above)
Comment (wajiw) [reply src/test/java/com/vailsys/freeclimb/webhooks/percl/GetSpeechActionCallbackTest.java:11]: Created a ticket to address this

**Comment by wajiw** [comment]
> @wajiw Do we need changelog updates in this?

Updated

**Review by ngarg-vail — APPROVED** [review]

**Pull request merged**